src/lib/createUser.js

```diff
@@ -2,6 +2,22 @@
 
 const { prompt } = require("inquirer");
 
+const checkIsUserExist = (users, newUser) => {
+  let isExist = false;
+  if (users && Array.isArray(users) && newUser) {
+    users.forEach(({ name, email, signingKey }) => {
+      if (
+        name === newUser.name &&
+        email === newUser.email &&
+        signingKey === newUser.signingKey
+      ) {
+        isExist = true;
+      }
+    });
+  }
+  return isExist;
```

Comment:
@awpogodin

If the **last index** in the `users array` is `false`, it appears to return **false** unconditionally.

How about using a [`some()`](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/some) `Higher-order function` rather than a `forEach()`?

```suggestion
return  users.some(({ name, email, signingKey }) =>
        name === newUser.name &&
        email === newUser.email &&
        signingKey === newUser.signingKey
    );
```

Reply:
I agree, it is preferable to use `some` here